### ISSUE TITLE
IPv6 single-stack with just link scope addressing

### ISSUE THREAD

**Issue by clauderobi** [containers/podman#17694]
### Feature request description

It would be useful, for security reasons to be able to setup some networks where only IPv6 is enabled (so single-stack, I know this is already possible) but where only IPv6 link scope addresses exist. In other words, do not assign any IPv6 address to the containers / pods.

Some containers / pods  would need to act as some sort of proxy and hence would be connected to a 2nd network but that beyond the scope of this enhancement request.

### Suggest potential solution

Simply add an option on "podman network create" to support IPv6 without defining a subnet. Currently, such command results in a failure with the ipv6 flag is present (which, BTW, is weird since it should be able to create an ULA subnet, but, this is also outside the scope of this request).

### Additional context

This would permit to fully isolate the containers / pods from any direct external access WITHOUT having to setup firewall rules, which may fail or get misconfigured at a later time. 

In other words, work in additive / addition mode rather than negative / removal mode. The former is much safer and greatly simplify devSecOps.

**Comment by Luap99** [comment]
> Simply add an option on "podman network create" to support IPv6 without defining a subnet. Currently, such command results in a failure with the ipv6 flag is present (which, BTW, is weird since it should be able to create an ULA subnet, but, this is also outside the scope of this request).

What is the error? Please provide the full commands.

If you do not want to assign ips you can use `--ipam-driver none` which I believe is what you asking for if I understand it correctly.
